### PR TITLE
Fix `F_1` implementation and allow for `g_5`

### PR DESCRIPTION
**Pull request by toonhasenack**
Define the observable `F1` as $2xF_1$.

### CODE REVIEW

**Comment by Radonirinaunimi** [comment]
For some reasons, the branch contained non-corresponding files again. I just re-wrote the history again.

We should also add a proper documentation for this.

**Comment by giacomomagni** [comment]
> We should also add a proper documentation for this.

Let me take care of it.

**Comment by giacomomagni** [comment]
Shall we add also `2xg_5` to complete the business?

**Comment by Radonirinaunimi** [comment]
> Shall we add also `2xg_5` to complete the business?

Yes, I think we should. It does not hurt in any case.

**Comment by giacomomagni** [comment]
> Yes, I think we should. It does not hurt in any case.

Thinking again, it requires some further modification as currently `xs_coeffs` combines only unpolarised SF

**Comment by Radonirinaunimi** [comment]
> > Yes, I think we should. It does not hurt in any case.
> 
> Thinking again, it requires some further modification as currently `xs_coeffs` combines only unpolarised SF

I also came to this exact realization last night. Let's leave this for a different PR $-$ something in the future.

**Review by Radonirinaunimi — APPROVED** [review]

**Comment by giacomomagni** [comment]
> I also came to this exact realization last night. Let's leave this for a different PR − something in the near future.

Ahaha ops, I gave a try in https://github.com/NNPDF/yadism/pull/236/commits/018a207d269687659f73a3fa6c5325c81f630f38, and in the end it was not that complicated, if you don't like it I can revert.

File: docs/source/theory/intro.rst [hunk 1]
```diff
@@ -84,25 +84,41 @@ given set of scattering bosons:
...
     it may be corrected (actually :math:`F_L` it's the object involved in
     Callan-Gross relation, for more information see :ref:`fl-corrections`)
 
-- Note that we compute :math:`xF_3` instead of the bare structure function to respect the native
-  scaling in the full cross section
-
 .. note::
 
-   Even :math:`F_1` is provided, but it is treated as a derived quantity, like
+   :math:`F_1` and :math:`2xg_5` are also provided as they are treated as a derived quantity, like
```
Comment (felixhekhorn) [inline docs/source/theory/intro.rst:121]:
```suggestion
   :math:`2xF_1` and :math:`2xg_5` are also provided as they are treated as a derived quantity, like
```

**Comment by Radonirinaunimi** [comment]
> > I also came to this exact realization last night. Let's leave this for a different PR − something in the near future.
> 
> Ahaha ops, I gave a try in [018a207](https://github.com/NNPDF/yadism/commit/018a207d269687659f73a3fa6c5325c81f630f38), and in the end it was not that complicated

Ah, perfect! Thanks!

> if you don't like it I can revert.

Nono, we can perfectly have it here, perhaps just need to rename the PR a bit.

**Comment by felixhekhorn** [comment]
> > I also came to this exact realization last night. Let's leave this for a different PR − something in the near future.
> 
> Ahaha ops, I gave a try in [018a207](https://github.com/NNPDF/yadism/commit/018a207d269687659f73a3fa6c5325c81f630f38), and in the end it was not that complicated, if you don't like it I can revert.

fine - keep it :upside_down_face:

**Review by felixhekhorn — APPROVED** [review]

**Pull request merged**